cypress/integration/SidePanelLabs.spec.ts

```diff
@@ -0,0 +1,82 @@
+import * as h from '../helpers';
+
+describe('Side Panel', () => {
+  before(() => {
+    h.stories.visit();
+  });
+
+  ['Default', 'AsAside', 'AsDiv'].forEach(story => {
+    context(`given the "${story}" story is rendered`, () => {
+      beforeEach(() => {
+        h.stories.load('Testing|React/Labs/Side Panel/Cypress', story);
+      });
+
+      const name = /Accessible Label Name/i;
+
+      it('should not have any axe errors', () => {
+        cy.checkA11y();
+      });
+
+      context('the button', () => {
+        it(`should have an accessible name`, () => {
+          cy.findByRole('button', {name}).should('exist');
+        });
+
+        it(`should have an aria-expanded attribute of 'true'`, () => {
+          cy.findByRole('button', {name}).should('have.attr', 'aria-expanded', 'true');
+        });
+
+        it(`should have an aria-controls attribute equal to the id of the panel`, () => {
+          cy.findByRole('button', {name}).then(button => {
+            const buttonAriaControlsValue = button.attr('aria-controls');
+            switch (story) {
+              case 'AsAside':
+                cy.findByRole('complementary', {name}).should(
+                  'have.attr',
+                  'id',
+                  buttonAriaControlsValue
+                );
+                break;
+              default:
+                cy.findByRole('region', {name}).should('have.attr', 'id', buttonAriaControlsValue);
+            }
+          });
+        });
+      });
+
+      it(`should have a panel with a landmark role`, () => {
+        switch (story) {
+          case 'AsAside':
+            cy.findByRole('complementary', {name}).should('exist');
+            break;
+          default:
+            cy.findByRole('region', {name}).should('exist');
+        }
+      });
+
+      context('when tabbing', () => {
```

Comment:
```suggestion
      context('when the tab key is pressed', () => {
```

Our context blocks explain the user's interaction with the browser. "when tabbing" isn't specific to what interactions is being performed - how many tab presses?